### ISSUE TITLE
Help text

### ISSUE THREAD

**Issue by Sadi58** [tubbadu/Scriptinator#4]
Help text is very useful indeed, but it's difficult to understand which icon is meant at the beginning, because there are 3 icons to be specified.  My guess is it's the "PeriodicScript" (the third) icon.
Also, here are some corrections/improvements I can suggest for the text - changes are in bold:

The icon will be set **every time** a command returns an output containing the following pattern:

```{PlasmoidIconStart}/home/path/to/icon.png{PlasmoidIconEnd}```

Accepted icon values are full paths (not relative paths) or icon names (`plasma`).

To change the **tooltip,** the process is the same:

```{PlasmoidTooltipStart}this is the new tooltip!{PlasmoidTooltipEnd}```
		
**NOTE: After** changing the preferences, if **they don't** apply immediately, just restart your session or run `plasmashell --replace` in a terminal **(or, better, in Krunner) without logging out.**

**Comment by tubbadu** [comment]
thanks for the suggestion! I'll update it as soon as possible!
It's true it may cause some confusion because of many different but similar functions, do you think a flow chart of the working process would be useful?

**Comment by Sadi58** [comment]
@tubbadu Exactly! A sort of flow chart of how this widget could work, and where users can make their own decisions (like skipping -not using - a step?) would make things much clearer I guess.
As I'm a freelance/retired editor/translator, I could help with the proofreading of the text if you like.

**Comment by tubbadu** [comment]
It would be greatly appreciated!!! You can make a pull request if you'd like, or just post here the changes and I'll apply them!

**Comment by Sadi58** [comment]
I have combined the current **Help** text with the contents of **README.md** in this file, which you can use in any way you like:
[https://docs.google.com/document/d/1rcg00PH_nB_x9kQ_kLjnFZYtseKDXCcrkMrq1uvVJG8](https://docs.google.com/document/d/1rcg00PH_nB_x9kQ_kLjnFZYtseKDXCcrkMrq1uvVJG8) 

Note: I removed the phrase "_until a new status is set_" from the text for the status value _attention_, because I've observed that it doesn't work, at least for now. It seems the plasmoid stops at the first instance of "`{PlasmoidStatusEnd}`" and doesn't read/execute anything after that.

**Comment by tubbadu** [comment]
I just updated the Help.qml page with a more polished and clear formatting, I will update the readme as well

thanks you very very much!

**Comment by Sadi58** [comment]
Looks great!
Just two minor issues: 1) section headings are bigger than the title **Help**, and 2) it could be better to make them (and also the code blocks) wrap like the other text as well.
